src/core/lightning/index.ts

```diff
@@ -451,17 +477,27 @@ export const LightningMixin = (superclass) =>
             throw new InsufficientBalanceError(undefined, { logger: lightningLogger })
           }
 
+          const price = await PriceService().getCurrentPrice()
+          if (price instanceof Error) throw price
+          const lnFee = toSats(fee)
+          const usd = sats * price
+          const usdFee = lnFee * price
+
           entry = await lockExtendOrThrow({ lock, logger: lightningLogger }, async () => {
             // reduce balance from customer first
-            const tx = await ledger.addLndPayment({
+            return LedgerService().sendLnTx({
```

Comment:
`send` makes sense in the app context but in this case you are adding records to the ledger.